jOOQ/src/main/java/org/jooq/Log.java

```diff
@@ -229,6 +239,84 @@
      */
     void error(Object message, Object details, Throwable throwable);
 
+    default boolean isEnabled(final Log.Level level) {
```

Comment:
We cannot have default methods in jOOQ yet as we're still supporting Java 6 and 7 in our commercial distributions. The additional API definitely makes sense, though.